webapp/src/workspace.ts

```diff
@@ -132,7 +132,7 @@ export function getHeaders(withDeleted = false) {
     let r = allScripts.map(e => e.header).filter(h =>
         (withDeleted || !h.isDeleted) &&
         !h.isBackup &&
-        !h.isSkillmapProject &&
+        !!h.isSkillmapProject === pxt.BrowserUtils.isSkillmapEditor() &&
```

Comment:
Are we showing the skillmap projects now in the home screen? I am not sure I understand this change.

Reply:
A comment would be great here.